### PR TITLE
chore: agent chart updates

### PR DESCRIPTION
**Pull request by yashmehrotra**

### CODE REVIEW

File: agent-chart/values.yaml [hunk 1]
```diff
@@ -1,14 +1,87 @@
 # Use this only if you want to replace the default that is .Chart.Name as the name of all the objects.
-nameOverride: "mc-agent"
```
Comment (moshloop) [inline agent-chart/values.yaml:2]: Can we add a `values.schema.json` ?

File: agent-chart/values.yaml [hunk 2]
```diff
@@ -1,14 +1,87 @@
...
+serviceAccount:
+  # Annotations to add to the service account
+  annotations: {}
+
+otel:
+  # OpenTelemetry gRPC collector endpoint in host:port format
+  collector: "{{.Values.global.otel.collector}}"
+  serviceName: "mission-control"
+
+smtp:
```
Comment (moshloop) [inline agent-chart/values.yaml:20]: This shouldn't be allowed

File: agent-chart/values.yaml [hunk 3]
```diff
@@ -1,14 +1,87 @@
...
+    key: DB_URL
+  jwtSecretKeyRef:
+    name: incident-commander-postgrest-jwt
+    key: PGRST_JWT_SECRET
+  storageClass:
+  storage: 20Gi
+  shmVolume: 256Mi
+  resources:
+    requests:
+      memory: 4Gi
```
Comment (moshloop) [inline agent-chart/values.yaml:83]: I think this should be down-scaled

File: agent-chart/values.yaml [hunk 4]
```diff
@@ -18,20 +91,28 @@ canary-checker:
...
-    storageClass: ""
-
+    runMigrations: false
+    embedded:
+      persist: false
+    external:
+      enabled: true
+      secretKeyRef:
+        name: incident-commander-postgres
+        key: DB_URL
```
Comment (moshloop) [inline agent-chart/values.yaml:115]: I don't think we should default the agentName - it should be empty and then mark as required in schema

File: agent-chart/values.yaml [hunk 5]
```diff
@@ -1,14 +1,87 @@
 # Use this only if you want to replace the default that is .Chart.Name as the name of all the objects.
-nameOverride: "mc-agent"
+image:
+  repository: docker.io/flanksource/incident-commander
+  pullPolicy: IfNotPresent
+  tag: "v0.0.439"
+
+global:
```
[the remark this answers is not in the document]
Comment (yashmehrotra) [reply agent-chart/values.yaml:7]: 👍🏽

File: agent-chart/values.yaml (hunk 2 above)
Comment (yashmehrotra) [reply agent-chart/values.yaml:20]: 👍🏽

File: agent-chart/values.yaml (hunk 3 above)
Comment (yashmehrotra) [reply agent-chart/values.yaml:83]: 👍🏽

File: agent-chart/values.yaml (hunk 4 above)
Comment (yashmehrotra) [reply agent-chart/values.yaml:115]: 👍🏽

**Comment by yashmehrotra** [comment]
@moshloop Have updated db params and values schema

**Pull request merged**